### PR TITLE
Mat 8569 button cql editor search result not working

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## MADiE PR

Jira Ticket: [MAT-8569](https://jira.cms.gov/browse/MAT-8569)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included
* [x] No extemporaneous files are included (i.e Complied files or testing results)
* [x] This PR is in to the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependancies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: src/CqlBuilderPanel/Includes/Results/IncludeResultActions.tsx [hunk 1]
```diff
@@ -18,37 +23,66 @@ const IncludeResultActions = ({
...
-            tw="mr-4"
-            onClick={() => {
-              onEdit(id, canEdit);
-            }}
-          >
-            View / Apply
-          </Button>
         </>
       )}
+      {operation === "edit" && (
```
Comment (sb-prateekkeerthi) [inline src/CqlBuilderPanel/Includes/Results/IncludeResultActions.tsx:57]: I didn't get this we are checking if the operation is edit and if yes we are doing viewing ? this looks odd

File: src/CqlBuilderPanel/Includes/Results/IncludeResultActions.tsx [hunk 2]
```diff
@@ -18,37 +23,66 @@ const IncludeResultActions = ({
...
+            onClick: () => onView(id),
+          }}
+        >
+          <CodeOffOutlinedIcon color="primary" />
+        </ToolTippedIcon>
+      )}
+      {!showDeleteAction && (
+        <Button
+          variant="outline"
+          data-testid={`edit-button-${id}`}
```
Comment (sb-prateekkeerthi) [inline src/CqlBuilderPanel/Includes/Results/IncludeResultActions.tsx:73]: data-testid={`edit-button-${id}`} --> says edit-button, but the operation that we are doing here is delete. I think this should be changed to data-testid={`delete-button-${id}`}

**Review by sb-prateekkeerthi — CHANGES_REQUESTED** [review]

File: src/CqlBuilderPanel/Includes/Results/IncludeResultActions.tsx [hunk 3]
```diff
@@ -18,37 +23,66 @@ const IncludeResultActions = ({
   onEdit,
   onDelete,
   onView,
+  operation,
 }: PropTypes) => {
   return (
     <Stack direction="row" alignItems="center">
       {canEdit && (
         <>
           {showDeleteAction && (
```
Comment (sb-prateekkeerthi) [inline src/CqlBuilderPanel/Includes/Results/IncludeResultActions.tsx:32]: {canEdit && showDeleteAction && ()} --> I think both conditions can be simplified into one

**Review by sb-prateekkeerthi — CHANGES_REQUESTED** [review]

File: src/CqlBuilderPanel/Includes/Results/IncludeResultActions.tsx [hunk 4]
```diff
@@ -18,37 +23,66 @@ const IncludeResultActions = ({
...
-              data-testid={`delete-button-${id}`}
-              disabled={false}
-              tw="mr-4"
-              onClick={() => {
-                onDelete(id);
-              }}
-            >
-              Delete
-            </Button>
+            <>
```
Comment (sb-prateekkeerthi) [inline src/CqlBuilderPanel/Includes/Results/IncludeResultActions.tsx]: {canEdit && showDeleteAction && ()} --> I think both conditions can be simplified into one

**Review by sb-prateekkeerthi — CHANGES_REQUESTED** [review]

**Review by sb-prateekkeerthi — APPROVED** [review]
